TRANSLATING.md

```diff
@@ -0,0 +1,225 @@
+* Either:
+  - Select **Clone a Repository from the Internet** and enter `yourusername/jamulus`, or
+  - Select your own fork of Jamulus from the list (`yourusername/jamulus`) and click the **Clone** button.
+* Select the Local Path where the project should be stored, and click **Clone**.
+* It will display the page **Cloning jamulus** with progress indication.
+* On completion of cloning, it will ask "How are you planning to use this fork?". Answer "To contribute to the parent project".
+
+### Qt Linguist
+
+Qt Linguist is a part of the Qt development suite, and may either be installed via your Operating System's packaging manager,
+or by downloading from the [Qt Open Source download page](https://www.qt.io/download-open-source). In the latter case you
+will need to create an account on the Qt website.
+
+Under Linux
+* On RedHat or CentOS: `yum install qt5-linguist` or `dnf install qt5-linguist`
+* On Debian or Ubuntu: `apt install qttools5-dev-tools`
+
+Instructions for use are in the [Qt Linguist Manual](https://doc.qt.io/qt-5/qtlinguist-index.html)
+
+----
+## Doing a translation
+
+### Get the most up-to-date files and set up a new branch
+
+#### With git command line tools
+
+The first step is to get the local repo up to date with the upstream master:
+
+```
+cd projectdir/jamulus
+git fetch upstream
+```
+
+This fetches information about the current state of the upstream repo. It is now necessary to apply any upstream changes to the local repo.
+As the user will not be updating the `master` branch themselves, `git rebase` can be used to fast-forward to the current state:
+
+```
+git checkout master
+git rebase upstream/master
+git push
+```
+
+Finally, create a new branch for the changes that will be done (do not just do them on `master`):
+
+```
+git checkout -b v370-german
+```
+
+(The branch name is chosen by the user; the above name v370-german is an example for the German translation of V3.7.0)
+
+The branch will be used later as the source of a Pull Request.
+
+#### With Github Desktop
+
+Select the current repository as `jamulus`, and the current branch as `master`.
+
+Click on "Fetch origin" to get fully up to date with upstream.
+
+Drop down the "Current Branch" menu and click on the **New Branch** button. Give the branch a name such as `v370-german` and click **Create Branch**.
+
+Do not click on **Publish branch** just yet.
+
+### Work on the translation file
+
+Open Qt Linguist, and navigate to the directory `src/res/translation` within your project directory.
+
+In this directory are translation source files for each language, each with a `.ts` suffix. Don't worry about the `.qm` files, as they are compiled when building the release code.
+
+Open the `.ts` file for the language being worked on.
+
+Each context in the left-hand column represents a source file or graphica form containing translatable strings. The icon beside each shows a green tick if there is nothing further to be done,
```

Comment:
Not a native speaker, but is "string(s)" a word which a non-technical person would read as what we mean? Should we introduce this?

Also, graphica**l** lacks an l, I think.

```suggestion
Each context in the left-hand column represents a source file or graphical form containing translatable words and phrases ("strings"). The icon beside each shows a green tick if there is nothing further to be done,
```